### PR TITLE
Ionesio's codebase cleanup 3

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by IonesioJunior** [comment]
@madhavajay @tudorcebere Could you guys please check if the deleted methods in the last commits aren't being used? I didn't find any other mention in PySyft codebase but they seems pretty important for the AST.

File: packages/syft/tests/syft/lib/util_test.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# syft absolute
+from syft.lib.util import full_name_with_name
+from syft.lib.util import full_name_with_qualname
+
+
+class TSTClass:
+    pass
+
+
+class FromClass:
```
Comment (rasswanth-s) [inline packages/syft/tests/syft/lib/util_test.py:11]: NITPICK: I think this class is not used.

File: packages/syft/tests/syft/lib/util_test.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @staticmethod
+    def static_method():
+        pass
+
+    def not_static_method(self):
+        pass
+
+
+class ToClass:
```
Comment (rasswanth-s) [inline packages/syft/tests/syft/lib/util_test.py:20]: NITPICK: This also is not used for testing.

**Review by rasswanth-s — APPROVED** [review]
LGTM

File: packages/syft/src/syft/lib/util.py [hunk 3]
```diff
@@ -19,6 +16,15 @@ def generic_update_ast(
     create_ast: Callable,
     ast_or_client: TypeUnion[Globals, AbstractNodeClient],
 ) -> None:
+    """
+    Update the abstract syntax tree data structure used by the Globals or AbstractNodeClient entities.
+
+    :param str lib_name:
```
Comment (rasswanth-s) [inline packages/syft/src/syft/lib/util.py:22]: MAYBE: change to Google Python style.

**Pull request merged**